_data/enterprise-landing-page-tabs-code.yml

```diff
@@ -0,0 +1,98 @@
+- id: eks
+  title: EKS cluster
+  content: |
+    <figure><figcaption>eks-cluster/main.tf</figcaption>
+      <pre>
+      <code class="language-hcl">
+        module "eks_cluster" {
+          source  = "github.com/gruntwork-io/service-catalog//eks-cluster"
```

Comment:
I suspect users are going to look at this generated very closely. Therefore, it's very important that it reflects the items they configured in step 1.

That is, there should be clear references to:

- A service mesh (Istio)
- CI / CD (Flux)
- Sending metrics to some tool (Prometheus)
- Auto scaling rules (min, max, CPU percent)
- Etc

That way, the user reading the page will see a clear, logical progression from one step to the next.

Reply:
This is a good point. I updated the core services module block to reflect the form selection from step 1.